src/com/oltpbenchmark/api/Worker.java

```diff
@@ -149,40 +137,20 @@ public final int getAndResetIntervalRequests() {
         return latencies;
     }
 
-    public final Procedure getProcedure(TransactionType type) {
-        return (this.procedures.get(type));
-    }
+    //public final Procedure getProcedure(TransactionType type) {
```

Comment:
Can we remove instead of commenting out?

Reply:
Done.